README.rst

```diff
@@ -0,0 +1,59 @@
+Vayesta Docs 
+===================
+
+This is the scaffold of the tutorials. To compile it, one needs only to type the following commands in the outer folder where the make files are.
+
+.. code-block:: console
+
+   [~]$ make clean
+   [~]$ make html
+
+The **html** results can be displayed via an internet browser (chrome, firefox, etc), following these lines of code
+
+.. code-block:: console
+
+   [~]$ cd build/html 
+   [~]$ firefox intro.html
+
+
+Cloning this branch:
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: console
+
+   git clone -b tutorials --single-branch git@github.com:BoothGroup/Vayesta.git
+
+Pushing into this branch:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+make changes, if you have used the -b tutorials then 
+
+.. code-block:: console
+
+   git remote show origin 
+
+must show the following result
+
+.. code-block:: console
+
+   * remote origin
+  Fetch URL: git@github.com:BoothGroup/Vayesta.git
+  Push  URL: git@github.com:BoothGroup/Vayesta.git
+  HEAD branch: master
+  Remote branch:
+    tutorials tracked
+  Local branch configured for 'git pull':
+    tutorials merges with remote tutorials
+  Local ref configured for 'git push':
+    tutorials pushes to tutorials (up-to-date)
+
+If this is **true**, then you dont need to do anything else of defining origins. Then, to push changes, you just need to do
+
+.. code-block:: console
+
+    git status  
+    git add --all -f  (sometimes the build directory is seen in the .gitignore)
+    git pull
+    git push
+
+and the changes will be done in the tutorials branch.
```

Comment:
Have you misplaced this file? It is in the root directory where there is already a `README.md`.

Reply:
No, this is just a general how to for internal use. You can remove it from the main branch, but I will strongly encourage to leave it, since it keeps a record of what should be done to create the spinx-apidocs from scratch.

Reply:
Ok - well it needs to go somewhere in the `docs` directory then, it shouldn't be in the root directory. We can have a subsection in the README.md about how to compile the docs, I can do that now. Plus, there is no need to mention git branches if everything will merge into master eventually - this isn't really user-facing info.

Reply:
In fact I'll hold off on mentioning the docs in the README.md until we host on gh-pages, that will make it a lot easier

Reply:
Since the documentation is under construction, I think we can have that README.rst in the apidocs branch. I have it also in my own local branch, but it is easier for somebody that is documenting parts of the code to know the procedure to build from scratch using sphinx-apidocs.

Reply:
Can you want to combine it with the docs/README.rst?

Reply:
Docstrings are displaying the description within classes and functons